torchbenchmark/util/model.py

```diff
@@ -29,4 +29,34 @@ def _set_mode(self, train):
         (model, _) = self.get_module()
         model.train(train)
 
-
+    def check_results(self):
```

Comment:
i'm wondering how this is intended to be used.   If it's used mainly locally by jit developers, it won't affect all users of the benchmark...  Should we make it slightly less intrusive by not needing to implement a 'check_results' override for models that don't jit?  is it sufficient to just try/except NotImplementedError inside this func to avoid the ones that aren't supported?